src/llfs/data_packer.test.cpp

```diff
@@ -189,4 +189,52 @@ TEST(DataPackerTest, ParallelDataCopyFalse)
   run_parallel_copy_test(false, 0);
 }
 
+void run_pack_record_test(const usize count)
+{
+  struct MyTemp {
+    int data;
+  };
+  const u64 mem_size = 64;
```

Comment:
Consider making this `constexpr` (compile-time constant); also, please use constant naming convention: `kMemSize`.

Reply:
sure

Reply:
Thanks!